src/lib/event_mapper/impl/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .general_aws_event_mapper import GeneralAwsEventMapper
```

Comment:
A bit confused with the folder namings - impl :)
Would propose to rename folders impl -> event_mapper (since actual event_mappers are there), and event_mapper -> smth describing product component (like event_service, event_component, etc.). I think it can correlate whith the notification_service structure.